src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -603,8 +614,12 @@ private async Task ValidateSignUpPoliciesAsync(Guid ownerId)
             UseSecretsManager = license.UseSecretsManager,
             SmSeats = license.SmSeats,
             SmServiceAccounts = license.SmServiceAccounts,
-            LimitCollectionCreationDeletion = !flexibleCollectionsMvpIsEnabled || license.LimitCollectionCreationDeletion,
-            AllowAdminAccessToAllCollectionItems = !flexibleCollectionsV1IsEnabled || license.AllowAdminAccessToAllCollectionItems
+            LimitCollectionCreationDeletion = license.LimitCollectionCreationDeletion,
+            AllowAdminAccessToAllCollectionItems = license.AllowAdminAccessToAllCollectionItems,
```

Comment:
I simplified this logic as it was too difficult to reason about and explain. It's much easier to just always update from the license file values. At the end of the day, `limitCollectionCreationDeletion` won't do anything unless `organization.FlexibleCollections == true` (which it won't unless we've turned on the migration and/or signup flag) and `allowAdminAccessToAllCollections` won't do anything unless the v1 flag is turned on. So let's just keep this logic simple and rely on the flags to control behaviour like we do everywhere else.